### PR TITLE
added cpp

### PR DESCRIPTION
**Pull request by Aparup123**
Added the code for `C++`. Also added the code for `C++ `in `compile.sh`, `run.sh` and `clean.sh` files

### CODE REVIEW

**Comment by Hxnter999** [comment]
How is this any different or any better than the C version of the code? at least take the time to rewrite using modern C++ features.

**Comment by Aparup123** [comment]
> How is this any different or any better than the C version of the code? at least take the time to rewrite using modern C++ features.

Sorry, i followed the same style cause then only we can compare the languages for the same code.

File: compile.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 clang -O3 c/code.c -o c/code
+clang++ cpp/code.cpp -o cpp/code
```
Comment (sergev) [inline compile.sh:2]: Flag ```-O3``` would help a little bit.

**Comment by bddicken** [comment]
Thanks for the contribution. Technically, C++ is a superset of C, and what we have here is basically just C code. Since it also uses the same complier (clang) this is basically just going to be the same thing. Therefore, I'm not going to include it.

Team C++ can just use the C scores.

**Pull request closed without merging**